common/src/address.rs

```diff
@@ -37,6 +37,7 @@ pub const COCKROACH_PORT: u16 = 32221;
 pub const CLICKHOUSE_PORT: u16 = 8123;
 pub const OXIMETER_PORT: u16 = 12223;
 pub const DENDRITE_PORT: u16 = 12224;
+pub const MGS_PORT: u16 = 12225;
```

Comment:
Completely arbitrary, just needs to not conflict with tfport and dendrite.

Reply:
... and wicket, I suppose

Reply:
I don't think this needs to change, but just a point of clarity - this is the port for the MGS dropshot server that nexus/wicketd talk to, but there is also an `MGS_PORT` constant internal to MGS that defines the UDP port it listens on on all the SP vlan links. My gut feeling is keeping `MGS_PORT` here for consistency with the other port numbers is fine, even if there's a different `MGS_PORT` constant defined somewhere else for the other side, but wanted to mention it in case y'all disagree.

Reply:
Ah, gotcha. Yeah, we can potentially rename these if they're used in the same context, but it sounds like there won't be any risk of overlapping (since they're using different protocols)